plugin/laravel.vim

```diff
@@ -101,23 +101,25 @@ augroup laravel_completion
   autocmd!
   " Set up nvim-completion-manager sources
   " :help NCM-source-examples
-  autocmd User CmSetup call cm#register_source({'name' : 'Laravel Route',
-        \ 'abbreviation': 'Route',
+  au User Ncm2Plugin call ncm2#register_source({
+        \ 'name' : 'Laravel Route',
```

Comment:
Horizontal space in the popup menu is at a premium. While it's nice to make the source of the completion explicit, I settled on omitting "Laravel". Thoughts?

```suggestion
        \ 'name' : 'Route',
```

Reply:
The `name` isn't in the pop up menu, the `mark` is. It appears like `mark` is the new key for `abbreviation` - which honestly confuses me. I'm guessing `mark` has something to do with internal syntax to vim's pop up menu?

Here are some screenshots:

![image](https://user-images.githubusercontent.com/744962/61583261-866c3200-ab03-11e9-8d2d-3ae3b21f7179.png)

![image](https://user-images.githubusercontent.com/744962/61583262-8a984f80-ab03-11e9-8235-d94934fc2ac0.png)

Reply:
Got it—my mistake. Looks good.